### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by Mqxx** [Earthcomputer/minimap-sync#2]
Hey I hav found this Mod on Modrinth and want to use it, but I dont know how 😅. Is there a documentaion or a wiki?

Currently I have installed the Mod on my Fabric server and my Fabric client.

I would be grateful for any help. Thank you very much!

**Comment by Earthcomputer** [comment]
Hi, you need to install it alongside a minimap mod on the client. VoxelMap is currently the best supported, others are in beta and you may experience issues with those currently. Then, your waypoints will automatically be shared with everyone else on the server who also has minimap sync on their clients, unless you set the waypoint to private. There is also the `/waypoint` command to interact with waypoints via commands.

**Comment by Mqxx** [comment]
Okay, so I need to install minimap-sync on both the client and server, for every user? I use Xaero Minimap and World Map

**Comment by Mqxx** [comment]
Seeing other players on the minimap works now. How can sync the waypoints global for every player?

**Comment by Earthcomputer** [comment]
It should happen automatically. You need the mod on all clients. However Xaero's support is in beta so expect bugs

**Comment by Mqxx** [comment]
Okay Thanks! Currently it does not work. I have installed Xaero Minimap and minimapsync on both the clients and the server but the only thing that gets synced are the players on the minimap. But not the waypoints. Are there some settings for the minimap or the minimap sync that I need to change?

**Comment by Earthcomputer** [comment]
Yeah that sounds like a bug to me then. I have only verified it to work properly on VoxelMap. I will look into it when I get the chance.

**Comment by Mqxx** [comment]
Thanks a lot! I really like this Mod. It connects all my friends on our server!
